bwresources.py

```diff
@@ -223,6 +223,7 @@ def _fill_mentions_params(self, data):
         filled["endDate"] = data["endDate"] if "endDate" in data else (
             datetime.date.today() + datetime.timedelta(days=1)).isoformat()
         filled["pageSize"] = data["pageSize"] if "pageSize" in data else 5000
+        filled["page"] = data["page"] if "page" in data else 0
```

Comment:
You can also do:
`filled["pageSize"] = data.get("page", 0)`
which I think just looks nicer, but it's obviously preference :)